### ISSUE TITLE
upgrade the tutorial cluster_permutation_timelock

### ISSUE THREAD

**Issue by yingjoeyzhou** [fieldtrip/website#203]
Section 1: **Between-trials experiments**

- [x] As described in issue #196 , _the starting point (preprocessing of the data) should be revised. This is copied from a separate tutorial, and should therefore also be revised there._ -> this has been addressed by Sophie in PR #206

- [x] Also, the output stat.posclusters(1) now contains prob, clusterstat, stddev, and cirange, instead of only prob and clusterstat. This part needs to be changed for sure. Maybe we want to add descriptions to these fields (i.e., what _stddev_ means)?

- [x] Consistency issue: cfg.statistic = 'indepsamplesT'; or cfg.statistic = 'ft_statfun_indepsamplesT' ?

- [x] For plotting, maybe add the option of using *ft_movieplotTFR*? -> WONTFIX for now (JM's executive decision)
- [x] In addition, the colormap (jet?) is outdated that needs an update.

**Comment by yingjoeyzhou** [comment]
Section 2: **Within-subjects experiments**

Similar issues for plotting apply.

**Comment by schoffelen** [comment]
It was a bit unfortunate that during the toolkit the majority of the changes that are referenced above were not in place. While the tutorial was running 'live' in the hands on session, I already updated the figures, to now contain the helmet layout and the parula colormap. For now I have changed the above comments to contain checkboxes, so that I will be able to tick them, when finishing this issue.

**Comment by schoffelen** [comment]
I have done a pass on the tutorial text, which now is a bit more in line with what we advocate elsewhere on the website (e.g. do not speak about 'significant clusters' etc)

Also, I cleaned up the formatting of the matlab code blocks.

- [ ] this still needs to be copied over to the related test_tutorial_* function, to check whether I did not break anything.

Also, with respect to the other still open action point: when loading the stat data from the ftp server, this file does not contain the mentioned cirange and stddev, so we'd probably want to update this to a more recent version, and then we can change the text in the tutorial as well. I will wait to do this until the fileserver is online again.
